lookml/customers.view.lkml

```diff
@@ -1,37 +1,24 @@
 view: customers {
   sql_table_name: analytics.customers ;;
 
+#------------------------------------------ PRIMARY KEY
+
   dimension: customer_id {
-    type: number
-    sql: ${TABLE}.customer_id ;;
     primary_key: yes
-    hidden: yes
-  }
-
-# ------------------------------------------------ Customer Info
-  dimension: first_name {
     type: string
-    sql: ${TABLE}.first_name ;;
+    sql: ${TABLE}.customer_id ;;
   }
 
-  dimension: last_name {
-    type: string
-    sql: ${TABLE}.last_name ;;
-  }
+#------------------------------------------ IDS
 
   dimension: email {
     type: string
     sql: ${TABLE}.email ;;
   }
 
-  dimension: phone {
-    type: string
-    sql: ${TABLE}.phone ;;
-  }
-
-# ------------------------------------------------ Dates
+#------------------------------------------ DATES
 
-  dimension_group: created {
+  dimension_group: created_date {
```

Comment:
I think you should have this field be `created` to keep consistent with removing `date` from your timestamps